test/render_jsonapi_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+class RenderTest < ActionController::TestCase
+  include Roar::Rails::TestCase
+
+  class SingersController < ActionController::Base
+
+    module SingerRepresenter
+      include Roar::JSON::JSONAPI
+      # type :musicians
+      property :name, :as => :title
+    end
+
+    module SingersRepresenter
+      include Representable::JSON::Collection
+      self.represenation_wrap = :musicians
+      items :extend => PersonRepresenter
+    end
+
+    include Roar::Rails::ControllerAdditions
+    include Roar::Rails::ControllerAdditions::Render
+
+    # NOTE: using :json_api will break
+    represents :json, :entity => SingerRepresenter, :collection => SingersRepresenter
```

Comment:
Are you sure you want that? Both entities and collections must be rendered with the "same" representer using JSONAPI - this is a media format specification and not up to you. So, there should be no need to declare the `:collection` representer here, this must be sorted internally.